app/containers/message/Reply.tsx

```diff
@@ -198,7 +182,7 @@ const Fields = React.memo(
 						<Text style={[styles.fieldTitle, { color: themes[theme].bodyText }]}>{field.title}</Text>
 						{/* @ts-ignore*/}
 						<Markdown
-							msg={field.value}
+							msg={field.value!}
```

Comment:
Same thing I commented on in the previous review.
Sure the value can be undefined?
However, you can see that in this case they did a treatment within the markdown in case the message was null.